src/Sarif.WorkItemFiling/WorkItemFilingMetadata.cs

```diff
@@ -14,14 +14,7 @@ public class WorkItemFilingMetadata
         public string Description { get; set; }
         public string AreaPath { get; set; }
         public List<string> Tags { get; set; }
-        public List<Attachment> Attachments { get; set; }
-
-        /// <summary>
-        /// The results that should be filed as a single work item.
-        /// </summary>
-        /// <remarks>
-        /// TEMPORARY. WHAT WE REALLY WANT HERE IS AN ATTACHMENT, A SARIF FILE WITH THE RELEVANT RESULTS.
-        /// </remarks>
-        public IList<Result> Results { get; set; }
+        public Attachment Attachment { get; set; }
+        public object Tag { get; set; }
```

Comment:
>public object Tag [](start = 8, length = 17)

this is a bit confusing since we also have a concept of tags in workitems and in fact have another properties "tags" in this object which have a very specific meaning. we should have a better name for this and/or provide a description to understand what exactly this object is.

Reply:
based on the usage, it appears this is the SarifFile, right? why not name it so?

---
In reply to: [308475219](https://github.com/microsoft/sarif-sdk/pull/1596#discussion_r308475219) [](ancestors = 308475219)

Reply:
I will rename this as Object. the point here is for this to be entirely decoupled from SARIF. :) it's very confusing to have 'Tag' and 'Tags' on the class, that was just an oversight.

---
In reply to: [308475568](https://github.com/microsoft/sarif-sdk/pull/1596#discussion_r308475568) [](ancestors = 308475568,308475219)